CMakeLists.txt

```diff
@@ -117,4 +117,19 @@ if (SIMSIMD_BUILD_SHARED)
     add_library(simsimd_shared SHARED ${SIMSIMD_SOURCES})
     target_include_directories(simsimd_shared PUBLIC "${PROJECT_SOURCE_DIR}/include")
     set_target_properties(simsimd_shared PROPERTIES OUTPUT_NAME simsimd)
+
+    install(
+        TARGETS simsimd_shared
+        ARCHIVE
+        BUNDLE CXX_MODULES_BMI
```

Comment:
Why do we need C++20 Modules in a C 99 library (with optional C 11 extensions)?

Reply:
You are right. I copied from my personal project and did not really care about the meaning. I personally just install everything.